en/finding-sorting-and-cleaning-entries/groups.md

```diff
@@ -1,132 +1,268 @@
+
+**%%%Help needed: Add an example of a sensible use here. %%%**
+
+### Mixing refining groups with including groups
+
+**%%%Help needed: Still true? If so, please, translate it using the current vocabulary?%%%**
+
+If a refining group is a subgroup of a group that includes its subgroups -- the refining group's siblings --, these siblings are ignored when the refining group is selected.
+
+### Types of groups
+
+JabRef has five types of groups:
+
+- **Explicit selection**.
+The group contains entries that were assigned manually.
+It behaves like a directory on disk, and contain only those entries that you explicitly assigned to it.
+
+- **Searching for a keyword**. 
+The group contain entries in which a certain field (e.g. `author`) contains a certain keyword (e.g. `Smith`).
+This method does not require manual assignment of entries, but uses information that is already present in the database.
 
-![groups having colors](../.gitbook/assets/groups-colors%20%281%29.png)
+- **Free search expression**.
+Like **Searching for a keyword**, but for several keywords in several fields.
 
-### Type of group
+- **Specified keywords**.
+**%%%Help needed: Please, add short description here %%%**
 
-{% hint style="warning" %}
-Outdated: The new interface \(JabRef 5.0\) displays a list of 5 choices under "Collect by"
-{% endhint %}
+- **Cited entries**.
+The group contains the entries cited in a LaTeX document, based on its *.aux* file.
 
-#### _Static_ groups
+#### Explicit selection
 
-Static groups are populated only by explicit manual assignment of entries. After creating a static group you select the entries to be assigned to it, and use either Drag and Drop or the main table's context menu to perform the assignment. To remove entries from a static group, select them and use the main table's context menu. There are no options to be configured.
+Groups based on explicit selection are populated only by manual assignment of entries.
 
-This method of grouping requires that all entries have a unique citation key. In case of missing or duplicate citation keys, the assignment of the affected entries cannot be correctly restored in future sessions.
+**%%% screenshot here: grab of the group type %%%**
 
-#### Dynamic groups
+After creating an explicit-selection group, you select the entries to be assigned to it, and use either drag-and-drop or the context menu **Add selected entries to this group** of the group interface.
 
-The content of a dynamic group is defined by a logical condition. Only entries that meet this condition are contained in the group. This method uses the information stored in the database itself, and updates dynamically whenever the database changes.
+To remove entries from an explicit-selection group, select them and use the context menu **Remove selected entries from this group** of the group interface.
 
-Two types of conditions can be used:
+**%%%Help needed: Relevant? True for this entry type only? %%%**
```

Comment:
Yes, you need to have uniquce citation keys. It's somewhat relevant as people were confused about it in the past. I would add it as an info-box (in gitbook).

Reply:
OK, I will add an infobox here, after merging, using gitbook.